{{cookiecutter.project_slug}}/.pre-commit-config.yaml

```diff
@@ -1,6 +1,5 @@
-exclude: "docs|node_modules|migrations|.git|.tox"
```

Comment:
pre-commit excludes `node_modules`, `.git` and `.tox` by default?

Reply:
reading description again I saw that apparently pre-commit respects .gitignore

Reply:
From my testing, it does. It should only check what's tracked in `.git` and these are already in our `.gitignore`.

Reply:
Do you want to take another look at this? I'm planning to merge it in a next couple of days if there is no objection